apps/portal-e2e/src/login/hacker/index.spec.ts

```
@@ -0,0 +1,39 @@
+import { actorCalled, Task } from '@serenity-js/core'
+import { beforeEach, describe, it } from '@serenity-js/playwright-test'
+import { BrowseTheWeb, Navigate } from '@serenity-js/web'
+import { loginNoTerms } from './abilities'
+
+export function startLoggedOut() {
+  return Task.where(
+    '#actor starts logged out',
+    Navigate.to('/login'),
+  )
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)